public/data/abubakar.json

```
@@ -0,0 +1,27 @@
+   {
+    "name":"Abubakar sadiq",
+    "type":"personal",
+    "bio": "open source enthusiast",
+    "avatar":"https://github.com/Abubakarsadiqq.png",
+    "links": [
+        {
+
+         "name":"follow me on github",
```

I recommend capitalizing the `h` and the `g`. What do you think?
```suggestion
         "name":"follow me on GitHub",
```